releasenotes-builder/src/main/java/com/github/checkstyle/Main.java

```diff
@@ -65,6 +68,25 @@ public static void main(String... args) {
                 final CliOptions cliOptions = cliProcessor.getCliOptions();
                 final Result notesBuilderResult = runGithubNotesBuilder(cliOptions);
                 errorCounter = notesBuilderResult.getErrorMessages().size();
+
+                final String version = cliOptions.getReleaseNumber();
+                final long numberOfPeriods = version.chars().filter(ch -> ch == '.').count();
+
+                Multimap<String, ReleaseNotesMessage> releaseNotes = notesBuilderResult.getReleaseNotes();
+                if (numberOfPeriods == 1) {
+                    if(releaseNotes.containsKey(Constants.BREAKING_COMPATIBILITY_LABEL)
+                        || releaseNotes.containsKey(Constants.NEW_FEATURE_LABEL)) {
```

Comment:
This is condition for correct version but you violate it.
Not good.
Please write Junit to keep stuff tested.

Reply:
I am not sure what you mean, how I am violating it.

Reply:
You increase error count

Reply:
Can you please help me, like JUnit in releasenotes-builder is quite tough as it doesn't have any previous JUnit for the same thing? Where do I have to make it?

Reply:
Look at https://github.com/checkstyle/contribution/blob/1b3e33600898a849791dc88c90df2bb3f80400e7/releasenotes-builder/src/test/java/com/github/checkstyle/templates/TemplateProcessorTest.java#L65

Just create copy and execute in your mode.

Reply:
Added

Reply:
if amount of dots is 1 and there is BREAKING_COMPATIBILITY_LABEL it should not be error. It is correct version.

Reply:
Changes Done

Reply:
> errorCounter = 0;

you can not change counter like this, you should skip increment.

Reply:
Changes Done